.mailmap

```diff
@@ -884,6 +884,7 @@ Maxence Mayrand <35958639+maxencemayrand@users.noreply.github.com>
 Mayank Raj <mayank_1901cs35@iitp.ac.in> mayank <54908605+mayankray2020@users.noreply.github.com>
 Mayank Raj <mayank_1901cs35@iitp.ac.in> mayank raj <mayank_1901cs35@iitp.ac.in>
 Mayank Singh <mayank.singh081997@gmail.com>
+Megan Ly <megan.ly@learnosity.com>
 Megan Ly <megan@artcompiler.com> Megan Ly <meganly@MacBook-Pro.local>
```

Comment:
@meganly there should not be two entries with different email addresses because this will mean that AUTHORS ends up listing you twice.

Which of these email addresses is the one that you actually want to have in the AUTHORS file?

Currently it is this:
https://github.com/sympy/sympy/blob/ed600afbc75001a95795a99667be8e2107bae905/AUTHORS#L835

Reply:
megan.ly@learnosity.com is my current email address and the one I would like listed.  My previous email was megan@artcompiler.com

Reply:
Okay so the .mailmap lines should look like this:
```
Megan Ly <megan.ly@learnosity.com> Megan Ly <megan@artcompiler.com>
Megan Ly <megan.ly@learnosity.com> Megan Ly <meganly@MacBook-Pro.local>
```
This is to say that the second person listed on the each line is an alias for the first person. Any commits recorded under the second name should be understood as referring to the same person as the first name. The AUTHORS file should only list the first name.

Can you open a PR to change those lines in .mailmap and also to update the line in AUTHORS?

Reply:
(Or if you plan to open another PR for any reason then it can be done at the same time)